react/features/base/config/reducer.js

```diff
@@ -194,6 +216,46 @@ function _translateLegacyConfig(oldValue: Object) {
         newValue.toolbarButtons = interfaceConfig.TOOLBAR_BUTTONS;
     }
 
+    const {
```

Comment:
looks a bit convoluted. Maybe something along the lines of 
`
Object.keys(oldValue)
  .filter(k => oldValue[k])
  .map(k => oldValue[k])
  .forEach(v => {
    ....
})`